content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -164,7 +165,7 @@ Here is an example of a valid YAML with all citation arguments:
                     nocite = c("@item1", "@item2")
                     ) %>%
       yml_toplevel(lang = "en-US",
-                   zotero = "false")
+                   zotero = "no")
```

Comment:
Results in:

![afbeelding](https://user-images.githubusercontent.com/19164640/124926285-a1405580-dffd-11eb-8a99-be5cf437c577.png)

Is this better than `'false'` ? I expect either both will be supported, or none. With regard to the quotation marks.

Reply:
I've done a little experiment (since I have Zotero here :wink:). Here are the results for current RStudio release 1.4.1717 in Linux Mint 20 - and this may not be stable in the future:

- inserting `zotero: true` gives the same result as not using the `zotero` key: support for Zotero is available.
- _anything else_ results in inactivating Zotero! This holds for `zotero: yes`, `zotero: 'yes'`, `zotero: 'true'`, `zotero: false`, `zotero: 'false'` and so on. So, your results so far would be effective in inactivating the Zotero link, but `'yes'` or `'true'` wouldn't...

Reply:
It is better. "no" is an alternative for FALSE. But 'FALSE' will be interpreted as a string.

Reply:
That's strange...